MountainArr_java_Pratheeksha

```diff
@@ -0,0 +1,35 @@
+import java.io.*;
+import java.lang.*;
+import java.util.*;
```

Comment:
Are these packages really required?
If yes, please update the imports for specific classes instead of importing complete package using `*`